string.c

```diff
@@ -11307,6 +11307,21 @@ rb_str_delete_suffix(VALUE str, VALUE suffix)
     return rb_str_subseq(str, 0, RSTRING_LEN(str) - suffixlen);
 }
 
+/*
+ *  call-seq:
+ *    ensure_suffix(suffix) -> self or new_string
+ *
+ *  :include: doc/string/ensure_suffix.rdoc
```

Comment:
Why separate the file?

Reply:
No particular reason. I just follow what I saw in other methods. I can in-line the doc if that's better

Reply:
It is for sharing the same description between *multiple* methods.